azure-pipelines.yml

```diff
@@ -597,6 +597,14 @@ jobs:
     - bash: echo "##vso[task.setvariable variable=bazelCacheGcpKeyPath]$(bazelCacheGcpKey.secureFilePath)"
       condition: eq(variables['Build.SourceBranchName'], 'master')
       displayName: Set the remote cache GCP key path
-    - bash: ci/scripts/quick-bazel.sh $(bazelCacheGcpKeyPath)
-      # The default timeout is too short to build verilator so let's skip it for the first pass
-      displayName: "Build and Test Software with Bazel (except //hw:verilator)"
+    - bash: |
+        # This command builds all software and runs all unit tests that run on
+        # the host. Additionally, it explicitly does not build Verilator,
```

Comment:
we do have a target //hw:verilator which is a verilated top_earlgrey, it's not really Verilator, which is confusing...